apis/config/v1alpha1/types_landscaper_config.go

```diff
@@ -25,6 +25,11 @@ type LandscaperConfiguration struct {
 	// CrdManagement configures whether the landscaper controller should deploy the CRDs it needs into the cluster
 	// +optional
 	CrdManagement *CrdManagementConfiguration `json:"crdManagement,omitempty"`
+	// DeployItemPickupTimeout defines how long a deployer can take to react on changes to a deploy item before the landscaper will mark it as failed.
+	// Time is specified in seconds. Set to -1 to disable timeout checking.
+	// Will be defaulted to 300 (5 minutes) if not specified.
+	// +optional
+	DeployItemPickupTimeout *int64 `json:"deployItemPickupTimeout,omitempty"`
```

Comment:
how about using a string and then parse it with `time.ParseDuration("1s")`.
With that we could use more human readable values. WDYT?

Reply:
Done :+1: